pkg/alerts/nrql_conditions.go

```diff
@@ -823,6 +823,7 @@ const (
       query
     }
     enabled
+	entityGuid
     description
```

Comment:
:memo: I am not sure why the diff looks this way. My IDE shows this is formatted well. 
![Screen Shot 2022-04-28 at 1 22 58 PM](https://user-images.githubusercontent.com/13971824/165839263-3b726ade-73b2-45dc-a16a-1f1b6000ceb0.png)